akka-http-tests/src/test/java/akka/http/javadsl/server/directives/PathDirectivesTest.java

```diff
@@ -294,11 +296,9 @@ public void testHexLongMatcher() {
 
     @Test
     public void testRestMatcher() {
-        PathMatcher<String> rest = PathMatchers.rest();
-
         TestRoute route =
             testRoute(
-                path("pets", rest).route(completeWithValueToString(rest))
+                path(Rest(), remainingPath -> complete(remainingPath))
```

Comment:
I saw a proposal somewhere while I was sick to rename `Rest` to `Remaining`, I'd like that :)